guides/source/active_record_migrations.md

```diff
@@ -589,14 +718,15 @@ add_foreign_key :articles, :authors
 ```
```

Comment:
What's the difference between add_reference plus foreign_key: true, and add_foreign_key?